frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/storage-tab-state-update.ts

```diff
@@ -97,11 +97,16 @@ export const internalStorageDiskBusUpdater = ({
         VMWizardStorageType.WINDOWS_GUEST_TOOLS,
       ].includes(type)
     ) {
-      const resultValidation = newValidations.validateBus(new DiskWrapper(disk).getDiskBus());
+      const resultValidation = newValidations.validateBus(
+        new DiskWrapper(disk).getType(),
+        new DiskWrapper(disk).getDiskBus(),
+      );
       if (!resultValidation.isValid && resultValidation.type === ValidationErrorType.Error) {
         someBusChanged = true;
         finalDisk = new DiskWrapper(disk, true)
```

Comment:
the main difference is that the final disk is mutated, so it needs to be copied to its own wrapper